packages/terra-form-fieldset/tests/wdio/fieldset-spec.js

```diff
@@ -10,4 +10,10 @@ Terra.describeViewports('Fieldset', ['tiny', 'medium'], () => {
 
     Terra.validates.element('populated Fieldset with hidden legend');
   });
+
+  it('should display populated Fieldset without legend and helptext', () => {
+    browser.url('/raw/tests/cerner-terra-core-docs/form-fieldset/fieldset-without-legend-and-helptext');
+
+    Terra.validates.element('populated Fieldset without legend and helptext');
```

Comment:
Screenshot is not generated for this test. Generate the screenshot OR if screenshot is not required then change the above function to 
```suggestion
    Terra.validates.accessibility();
```